tests/test_browser_custom_keybinds.py

```
@@ -16,11 +16,42 @@
+
+def v2_call(client: cmux, method: str, params: dict[str, Any] | None = None, request_id: str = "1") -> dict[str, Any]:
+    payload = {
+        "id": request_id,
+        "method": method,
+        "params": params or {},
+    }
+    raw = client._send_command(json.dumps(payload))
+    try:
+        parsed = json.loads(raw)
+    except json.JSONDecodeError as exc:
+        raise RuntimeError(f"Invalid v2 JSON response for {method}: {raw}") from exc
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)